templates/createDropbox.hbs

```diff
@@ -75,7 +75,9 @@
 
     <div class="form-group">
       <label for="dob">Child's date of birth</label>
-      <input type="date" class="form-control" id="dob" name="dob" required />
+      <input type="date" class="form-control" id="dob" name="dob" placeholder="dd/mm/yyyy"
+        pattern="(^(((0[1-9]|1[0-9]|2[0-8])[\/](0[1-9]|1[012]))|((29|30|31)[\/](0[13578]|1[02]))|((29|30)[\/](0[4,6,9]|11)))[\/](19|[2-9][0-9])\d\d$)|(^29[\/]02[\/](19|[2-9][0-9])(00|04|08|12|16|20|24|28|32|36|40|44|48|52|56|60|64|68|72|76|80|84|88|92|96)$)"
```

Comment:
wow, you really went to town on it huh. leap years and everything lolol

Reply:
what message does the validator give you if it fails?

Reply:
"Match the requested format"